openlibrary/templates/search/work_search_facets.html

```diff
@@ -1,15 +1,14 @@
-$def with (facet_map, param, facet_counts=None)
+$def with (param, facet_counts=None, async_load=True, path=None, query={})
```

Comment:
What's the difference between `param` and `query`?

Reply:
`query` is a dictionary containing all parsed query parameters.
`param`, defined [here](https://github.com/internetarchive/openlibrary/blob/04fb777459d99c4e1010f46af51fc37d98df1e41/openlibrary/plugins/worksearch/code.py#L477), contains a limited set of facet fields.